pom.xml

```
@@ -100,7 +100,7 @@
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-2.387.x</artifactId>
-        <version>2278.v47b_4508e256a</version>
+        <version>2496.vddfca_753db_80</version>
```

Final release of bom-2.387.x
```suggestion
        <version>2543.vfb_1a_5fb_9496d</version>
```